modules/lambdas/main.tf

```diff
@@ -298,6 +298,75 @@ resource "aws_lambda_function" "request_status_for_granule" {
   }
 }
 
+# API Gateway
+resource "aws_api_gateway_rest_api" "request_status_for_granule_api" {
+  name = "${var.prefix}_request_status_for_granule_api"
+}
+
+resource "aws_api_gateway_resource" "request_status_for_granule_api_resource" {
+  path_part   = "{proxy+}"
+  parent_id   = aws_api_gateway_rest_api.request_status_for_granule_api.root_resource_id
+  rest_api_id = aws_api_gateway_rest_api.request_status_for_granule_api.id
+}
+
+resource "aws_api_gateway_method" "request_status_for_granule_api_method" {
+  rest_api_id   = aws_api_gateway_rest_api.request_status_for_granule_api.id
+  resource_id   = aws_api_gateway_resource.request_status_for_granule_api_resource.id
+  http_method   = "ANY"
+  # todo: Make sure this is locked down against external access.
```

Comment:
I know we wanted to make use of Cumulus' security, but I don't know if we have a task for that or if things have to change here.

Reply:
From what I recall, our API would be private and only accessible to the Cumulus API. See the notes from https://wiki.earthdata.nasa.gov/pages/viewpage.action?pageId=202211487

I believe Cumulus will handle all the authorization so initially we should be able to have this as None.